client-web/src/functional-e2e/test-plan-applications.md

```diff
@@ -0,0 +1,897 @@
+
+**Steps:**
+
+1. Sign in as `${TestUser.SPACE_ADMIN}@alkem.io`
+2. Navigate to Level 0 Space → Settings → Community
+3. Locate application from `${TestUser.NON_SPACE_MEMBER}@alkem.io`
+4. Click "Reject" button
+5. Confirm rejection if confirmation dialog appears
+
+**Expected Results:**
+
+- Reject button is visible and clickable
+- Confirmation dialog may appear
+- After rejection, application status changes to "Rejected"
+- Application is removed from pending list or marked as rejected
+- Success notification appears
+- Applicant receives notification of rejection
+
+#### 10.2 Verify Rejected User Does Not Gain Access
+
+**Steps:**
+
+1. Admin rejects application for Level 1 Subspace
+2. Sign out and sign in as `${TestUser.NON_SPACE_MEMBER}@alkem.io`
+3. Navigate to Level 1 Subspace
+
+**Expected Results:**
+
+- Lock icon still displayed
+- User cannot access space content
+- User is not in members list
+- Apply button may be available again or shows "Application Rejected"
+
+#### 10.3 Rejection Notification Content
+
+**Steps:**
+
+1. Admin rejects application
+2. Sign in as applicant
+3. Click bell icon
+4. View rejection notification
+
+**Expected Results:**
+
+- Notification clearly states rejection
+- Space name is included
+- Timestamp is shown
+- User understands application was rejected
+
+---
+
+### 11. Application Review - Delete Applications
+
+**Seed:** `./client-web/src/functional-e2e/seed-applications.spec.ts`
+
+#### 11.1 Delete Application
+
+**Steps:**
+
+1. Sign in as `${TestUser.SPACE_ADMIN}@alkem.io`
+2. Navigate to Level 0 Space → Settings → Community
+3. Locate an application
+4. Click "Delete" button
+5. Confirm deletion if confirmation dialog appears
+
+**Expected Results:**
+
+- Delete button is visible
+- Confirmation dialog appears warning about permanent deletion
+- After deletion, application is completely removed from list
+- No notification is sent to applicant (or system notification is sent)
+- Action is irreversible
+
+#### 11.2 Verify Deleted Application Cannot Be Recovered
+
+**Steps:**
+
+1. Admin deletes an application
+2. Refresh the page
+3. Search for the deleted application
+
+**Expected Results:**
+
+- Deleted application does not appear in any list
+- Application data is permanently removed
+- No way to restore the application
+
+---
+
+### 12. Application Management - Bulk Operations
+
+**Seed:** `./client-web/src/functional-e2e/seed-applications.spec.ts`
+
+#### 12.1 Multiple Applications Display
+
+**Steps:**
+
+1. Have 3+ applicants apply to Level 0 Space
+2. Sign in as `${TestUser.SPACE_ADMIN}@alkem.io`
+3. Navigate to Level 0 Space → Settings → Community
+4. View applications list
+
+**Expected Results:**
+
+- All applications are displayed in a list or table
+- Each application shows applicant name
+- Application date/time is shown
+- Actions (View, Approve, Reject, Delete) are available for each
+- List is sortable or filterable
+
+#### 12.2 Mixed Actions on Multiple Applications
+
```

Comment:
Data could be generated with api.
No need to validate notifications, as this is done as part of above scenarios.